.github/workflows/bitrise-action.yml

```diff
@@ -7,8 +7,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
```

Comment:
If we don't package the app on the action level then we should add a GitHub pre-hook to run the `yarn build` before committing.

Reply:
Yes totally agree, I'll add this